src/man/sssd-ifp.5.xml

```diff
@@ -55,6 +55,11 @@
                             responder. User names are resolved to UIDs at
                             startup.
                         </para>
+                        <para>
+                            The user should be a local one, i.e. accessible via
```

Comment:
"Every user"?
Better to ask @justin-stephenson or @spoore1 or @sidecontrol to proof read.

Reply:
Something like "User names provided must be local", or "Local user names are required" sounds better IMO.